bin/sl-run.js

```diff
@@ -18,7 +18,14 @@ var agentOptions = {
   logger: config.logger,
 };
 
-switch (config.profile) {
+var tracer = require('concurix');
+var tracerOptions = {
+  archiveInterval: 20000,
+  accountKey: agent().config.appName,
+  useHttp: false
+};
+
+switch(config.profile) {
```

Comment:
I think i did once and reverted it because that's not the code area I touched this time.  in any case, added it.